CODEOWNERS

```diff
@@ -5,9 +5,163 @@
 *       @openhab/2-x-add-ons-maintainers
 
 # Add-on maintainers:
+/addons/binding/org.openhab.binding.airquality/ @kubawolanin
+/addons/binding/org.openhab.binding.airvisualnode/ @3cky
+/addons/binding/org.openhab.binding.allplay/ @dominicdesu
+/addons/binding/org.openhab.binding.amazondashbutton/ @OLibutzki
+/addons/binding/org.openhab.binding.amazonechocontrol/ @mgeramb
+/addons/binding/org.openhab.binding.atlona/ @tmrobert8
+/addons/binding/org.openhab.binding.autelis/ @digitaldan
+/addons/binding/org.openhab.binding.avmfritz/ @cweitkamp
+/addons/binding/org.openhab.binding.avmfritz.test/ @cweitkamp
+/addons/binding/org.openhab.binding.bigassfan/ @mhilbush
+/addons/binding/org.openhab.binding.boschindego/ @jofleck
+/addons/binding/org.openhab.binding.chromecast/ @kaikreuzer
+/addons/binding/org.openhab.binding.chromecast.test/ @kaikreuzer
+/addons/binding/org.openhab.binding.cm11a/ @BobRak
+/addons/binding/org.openhab.binding.coolmasternet/ @projectgus
 /addons/binding/org.openhab.binding.deconz/ @davidgraeff
-/addons/binding/org.openhab.binding.enocean/ @fruggy83
+/addons/binding/org.openhab.binding.denonmarantz/ @jwveldhuis
+/addons/binding/org.openhab.binding.dlinksmarthome/ @MikeJMajor
+/addons/binding/org.openhab.binding.dscalarm/ @RSStephens
 /addons/binding/org.openhab.binding.dsmr/ @Hilbrand
+/addons/binding/org.openhab.binding.dsmr.test/ @Hilbrand
+/addons/binding/org.openhab.binding.elerotransmitterstick/ @vbier
+/addons/binding/org.openhab.binding.enocean/ @fruggy83
+/addons/binding/org.openhab.binding.evohome/ @Nebula83
+/addons/binding/org.openhab.binding.exec/ @kgoderis
+/addons/binding/org.openhab.binding.feed/ @svilenvul
+/addons/binding/org.openhab.binding.feed.test/ @svilenvul
+/addons/binding/org.openhab.binding.feican/ @Hilbrand
+/addons/binding/org.openhab.binding.folding/ @fa2k
+/addons/binding/org.openhab.binding.freebox/ @clinique
+/addons/binding/org.openhab.binding.fronius/ @trokohl
+/addons/binding/org.openhab.binding.ftpupload/ @paulianttila
+/addons/binding/org.openhab.binding.gardena/ @paulianttila
```

Comment:
Gardena binding owner should be @gerrieg not me?

Reply:
I'll correct that! Copy/paste is always an issue on Linux desktops. :wink: